walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,59 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> times = new ArrayList<TimeRange>();
+    int beginning = TimeRange.START_OF_DAY;
+    int end = TimeRange.END_OF_DAY;
+    TimeRange middle_current;
```

Comment:
The naming is a bit confusing. Can you change it?
(I am not good at naming either so no suggestions :p)

Reply:
Or you can move your comments below here...